client/my-sites/checkout/composite-checkout/components/checkout-help-link.tsx

```diff
@@ -193,22 +174,18 @@ export default function CheckoutHelpLink() {
 		);
 	};
 
-	// If chat is available and the cart has a pre-sales plan or is already eligible for chat.
-	const shouldRenderPaymentChatButton =
-		happyChatAvailable && ( isPresalesChatEligible || supportVariation === SUPPORT_HAPPYCHAT );
+	const zendeskPresalesChatKey: string | false = config( 'zendesk_presales_chat_key' );
+	const isPresalesZendeskChatEligible = presalesZendeskChatAvailable && isEnglishLocale;
```

Comment:
It looks like with this code, you'd be taking the pre-sales chat option away from non-English speakers.  Currently with Spanish, for example, I do see a chat option:

![pre-sales-chat](https://user-images.githubusercontent.com/235183/203152032-0e65afa4-6240-4b3e-ac87-c2e7484fd765.png)

I'm not sure if the `isEnglishLocale` check is permanent or temporary, but either way, Is there a reason not to leave the old option around as a backup for when Zendesk chat isn't available?

Reply:
> It looks like with this code, you'd be taking the pre-sales chat option away from non-English speakers.

AFAIK, that's the scope of the test for the Zendesk chat widget.
However, if the user is on a non-English locale, they do (from my testing and understanding of the code) get the normal pre-sales chat via Happy Chat option:
<img width="363" alt="Screenshot 2022-11-25 at 13 23 59" src="https://user-images.githubusercontent.com/3392497/203996281-32a58917-eb7e-4ab9-a0bc-a7321cbb1097.png">

Clicking on it goes to Help Center and from there you can contact support (email or chat). Confirmed it's the same for the old contact form (FAB/inline-form):

<img width="598" alt="Screenshot 2022-11-25 at 13 36 05" src="https://user-images.githubusercontent.com/3392497/203997080-5e1aac74-4b7b-43f7-a857-3fe5a351afd6.png">

And the same on production.

Let me know if I misunderstood and you meant a different use case 🙇

Reply:
>Clicking on it goes to Help Center and from there you can contact support (email or chat).

That's only if you're **already** a paid customer.  If you're a new customer who hasn't bought anything yet, the Help Center only gives you access to the forums.  Whereas the link shown in my screenshot above is a pre-sales chat option that (when it appears) is currently made available even to users who haven't purchased anything yet.  So that's the thing that this pull request seems like it would be taking away from non-English users.

Reply:
This pull request is definitely taking pre-sales chat away from **some** users who currently have it -- that's how I was able to make my screenshot above (that's a user who would lose pre-sales chat after this pull request is merged).

It is definitely possible there is a test going on and some users who are in the test have already lost access to pre-sales chat separately, though... I didn't think about that :smile:  It does seem like I see different behavior on this page with two of my test accounts, and perhaps that's the reason why.